conda_recipe_manager/commands/bump_recipe.py

```diff
@@ -15,11 +15,50 @@
 from conda_recipe_manager.parser.recipe_parser import RecipeParser
 from conda_recipe_manager.types import JsonPatchType
 
-
 # TODO Improve. In order for `click` to play nice with `pyfakefs`, we set `path_type=str` and delay converting to a
 # `Path` instance. This is caused by how `click` uses decorators. See these links for more detail:
 # - https://pytest-pyfakefs.readthedocs.io/en/latest/troubleshooting.html#pathlib-path-objects-created-outside-of-tests
 # - https://github.com/pytest-dev/pyfakefs/discussions/605
+
+
+def get_required_patch_blob(recipe_parser: RecipeParser, build_num: bool) -> JsonPatchType:
+    """
+    Returns the required JSON Patch Blob
+
+    :recipe_parser: RecipeParser object
+    :build_num: `build_num` boolean flag
+    :returns: A JSON Patch blob to add or modify the build number
+    """
+
+    # try to get "build" key from the recipe, exit if not found
+    try:
+        recipe_parser.get_value("/build")
+    except KeyError:
+        print_err("`/build` key could not be found in the recipe.")
+        sys.exit(ExitCode.ILLEGAL_OPERATION)
+
+    # if build key is found, try to get build/number key
+    # in case of `build_num` set to false, `build/number` key will be added and set to zero
+    # when `build_num` is set to true, throw error and sys.exit()
+    try:
+        build_number = recipe_parser.get_value("/build/number")
+        required_patch_blob = cast(JsonPatchType, {"op": "replace", "path": "/build/number", "value": 0})
+        if build_num:
+            if not isinstance(build_number, int):
+                print_err("Build number is not an integer.")
+                sys.exit(ExitCode.ILLEGAL_OPERATION)
+            required_patch_blob = cast(
+                JsonPatchType, {"op": "replace", "path": "/build/number", "value": build_number + 1}
+            )
+    except KeyError:
+        required_patch_blob = cast(JsonPatchType, {"op": "add", "path": "/build/number", "value": 0})
```

Comment:
Similar to above, I would use the return-early paradigm to skip the setting of a variable.